simulator/fault_injection/src/fault_injection_node/fault_injection_node.cpp

```diff
@@ -39,8 +39,6 @@ namespace fault_injection
 {
 #ifdef USE_DEPRECATED_TO_YAML
 using rosidl_generator_traits::to_yaml;
-#else
-using tier4_simulation_msgs::msg::to_yaml;
```

Comment:
Since we can use `ADL`, I removed it.
https://en.cppreference.com/w/cpp/language/adl